testbeam_analysis/dut_alignment.py

```diff
@@ -296,35 +299,37 @@ def prealignment(input_correlation_file, output_alignment_file, z_positions, pix
                     background = np.array(background, dtype=np.int)  # make Numpy array
                     data = (data - background).astype(np.int)  # remove background
                     data -= data.min()  # only positive values
+ 
```

Comment:
This calculations take quite some time, and the line fit usually does not fail due to missing start parameters. Or are the mu/sigma parameters for the Gaussian fits set by the hough results? I have a tough time reading the code on github because the line length is too long and scrolling is not supported :-(.